packages/main/src/Dialog.js

```diff
@@ -77,75 +84,75 @@ class Dialog extends Popup {
 	}
 
 	static get styles() {
-		return [Popup.styles, dialogCss];
+		return [dialogCSS, popupCSS];
+	}
+
+	constructor() {
+		super();
+
+		// mark dialog for registry as it is always modal
+		this.modal = true;
+		this.onPhone = isPhone();
+	}
+
+	reposition() {
+		this.style.display = "inline-block";
```

Comment:
When I search for "this.styles.display" in the main package, there are multiple hits in all popup/dialog/popover/resp. popover files. Please create 2 functions on Popup.js level with appropriate names, one setting to `none`, the other to `inline-block` and call these from all descendants

Reply:
I added `show()` and `hide()` protected methods